BotBrownCli/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "BotBrownCli": {
       "commandName": "Project",
-      "commandLineArgs": "-notwitch -debug"
+      "commandLineArgs": "-notwitch -debug -ccp:..\\..\\..\\..\\Configurations -csp:..\\..\\..\\..\\Sounds"

```

Comment:
Standard debug profile with notwitch?

Reply:
Yes. The idea was that you usually only test stuff that does not need that level of integration